src/rngs/mock.rs

```diff
@@ -75,6 +76,18 @@ impl RngCore for StepRng {
     }
 }
 
+impl SeedableRng for StepRng {
+    type Seed = [u8; 16];
+
+    fn from_seed(seed: Self::Seed) -> Self {
+        let seed_u64: [u64; 2] = transmute!(seed);
+        StepRng {
+            v: seed_u64[0],
+            a: seed_u64[1],
```

Comment:
Concern: if this were to randomly sample a 0, the output would be constant; if it randomly samples an even number, half the output values will be unreachable. I'm not entirely sure what behaviour people would expect here but suggest we do `seed_u64[1] | 1`.

Reply:
This makes me thing this impl is a bad idea. The usecases for `StepRng` are probably the easy predictability and `SeedableRng` goes against this

Reply:
It may be that my concern is entirely invalid given that this is a *mock* RNG, but you may also be right.